website/content/docs/nia/installation/requirements.mdx

```diff
@@ -59,21 +59,25 @@ If you would like to run a Consul cluster rather than a single agent, refer to [
 
 Consul-Terraform-Sync integrations for the Terraform driver utilizes Terraform providers as plugins to interface with specific network infrastructure platforms. The Terraform driver of Consul-Terraform-Sync inherits the expansive collection of Terraform providers to integrate with, and with release of [Terraform 0.13](https://www.hashicorp.com/blog/announcing-hashicorp-terraform-0-13/), this extends to include providers written by the community too by using [provider source](https://www.hashicorp.com/blog/adding-provider-source-to-hashicorp-terraform/).
 
-### Finding and Using Terraform Providers
+### Finding Terraform Providers
 
 To find providers for the infrastructure platforms you use, browse the providers section of the [Terraform Registry](https://registry.terraform.io/browse/providers).
 
+Continue to the next page to [get started with configuring Consul-Terraform-Sync and how to use Terraform providers and modules for tasks.](/docs/nia/installation/configure)
+
```

Comment:
This sentence gives me the sense that if the user has found the Terraform Provider, they can continue on to the next page and the below content isn't relevant. I'm not sure if that's the intention? Let me know if I'm misunderstanding. For me, I think the "Network Integration (using a Terraform Module)" would be useful for users to learn briefly about modules and maybe use a partner module, even if users have found a provider

Reply:
That's true i had copied it here as well because the section was originally named "Finding and Using Terraform Providers" but didn't go into how to use them. I will remove this thanks